Artifacts/windows-visualstudio2015/Artifactfile.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+  "title": "Visual Studio 2015",
+  "description": "Installs Visual Studio 2015",
+  "publisher": "Microsoft",
+  "tags": [
+    "Windows"
+  ],
+  "iconUri": "https://i3-vso.sec.s-msft.com/dynimg/IC806106.png",
+  "targetOsType": "Windows",
+  "parameters": {
+    "sku": {
+      "type": "string",
+      "displayName": "SKU",
+      "description": "The Visual Studio SKU to install.",
+      "allowedValues": [
+        "Enterprise",
+        "Professional"
+      ]
+    },
+    "installerArgs": {
+      "type": "string",
+      "defaultValue": "",
+      "allowEmpty": true,
+      "displayName": "Optional args passed to VS installer",
+      "description": "Additional arguments to pass when installing VS."
+    }
+  },
+  "runCommand": {
+    "commandToExecute": "[concat('powershell.exe -ExecutionPolicy bypass \"& ./artifact-main.ps1', ' -sku ', parameters('sku'), ' -installerArgs ', '''', parameters('installerArgs'), '''', '\"')]"
```

Comment:
Any chance the `sku` could contains spaces? If so, you may want to wrap it around quotes like you do for `installerArgs`.

Reply:
I'll wrap it in quotes. The artifact param has an allowedValues set and nothing contains spaces there today, but it's safer in quotes.